src/Broadcaster.cpp

```diff
@@ -171,6 +171,19 @@ void cWorld::BroadcastBlockEntity(Vector3i a_BlockPos, const cClientHandle * a_E
 
 
 
+void cWorld::BroadcastBossBarUpdateHealth(UInt32 a_UniqueID, float a_FractionFilled)
+{
+	ForClientsInWorld(*this, nullptr, [&](cClientHandle & a_Client)
```

Comment:
I don't think we want to send the bar to everybody

Reply:
Just with the loaded chunks or smth - like before

Reply:
Oh... Hm... This is more complicated. We don't habe a entity to attach to so no position to determine if it should be sent

Reply:
We could do a optional parameter with coordinates that is hidden from the API so that we can send it just for some chunks.

It seems like the vanilla implementation displays the non entity bound bar to everyone

Reply:
If it's only being called by cWither & cEnderDragon adding a cEntity & parameter should work.

Reply:
this also applies for all other new Broadcast Methods. So there needs to be a solution for all methods

Reply:
We only added one (server internal) broadcast